packages/transition-frontend/webpack.config.js

```diff
@@ -173,8 +191,11 @@ module.exports = (env) => {
       )
     ],
     resolve: {
+      mainFields: ['browser', 'main', 'module'],
       modules: ['node_modules'],
-      extensions: ['.json', '.js', '.css', '.scss', '.ts', '.tsx'],
+      // extensions: ['.json', '.js', '.css', '.scss', '.ts', '.tsx'],
```

Comment:
Any reason to keep this commented out? 
(I don't know anything about this, I guess there's a reason to drop the css)

Reply:
I should have kept notes... From memory, docs mentionned to be explicit when importing (don't presume .css extension) for better performance.